### PR TITLE
ft-supporting multiple languages

### PR DESCRIPTION
**Pull request by JPaul23**
# Chore-Supporting Multiple Language
## Overview
In this feature, I have successfully completed the **setup of supporting multiple languages**. 
Those include: 
- English (en) as default of our web
- French (fr) as secondary <-- requested by Product Owner
- Kinyarwanda (rw) as secondary
## Testing
To test this follow these steps:
1. pull this branch
2. run `npm install` this will install all dependencies in `package.json`
3. run `npm run build` to build the app with _babel_
4. run `npm start` to start the server.
5. navigate to **`http://localhost:<PORT>/ `** the PORT will depend to your defined PORT in .ENV file

### Naive Test Result
These are the results from test done in Postman. To use this approach in web browser you have to change the link to 
**`http://localhost:<PORT>/ ?lng=<LNG>`**  where LNG is the language index **[en, fr, rw** ].
_example:_ localhost:5000/?lng=rw for the Kinyarwanda translation.
#### English support
<a>test</a>
#### Kinyarwanda support
<a>naive-test</a>
#### French support
<a>naivefr</a>

### Automated testing
I used [Mocha](https://mochajs.org/) , [Chai,](https://www.chaijs.com/) , [Istanbul](https://istanbul.js.org/) A library for testing code coverage. The result are an image below
<a>test</a>

**Happy Reviewing!!** ⌨️🖱️

### CODE REVIEW

**Comment by Julesbi** [comment]
### This is supporting multiple languages as expected

This is for Kinyarwanda





And this  for French

File: test/app.spec.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import chai, { expect } from 'chai'
```
Comment (NyiingangoPacifique) [inline test/app.spec.js]: You see you are using different test file one is test.js which is from our develop and then app.spec.js 
you can continue using test.js

Comment (JPaul23) [reply test/app.spec.js]: let me do that. Sorry forgot to delete the app.spec.js

**Review by David-Neza — COMMENTED** [review]
PR reviewed successfully --everything is good!!

File: test/app.spec.js (hunk 1 above)
Comment (NyiingangoPacifique) [reply test/app.spec.js]: Now working thanks

**Review by NyiingangoPacifique — APPROVED** [review]

File: test/test.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    it('welcomes user to the api in french', (done) => {
+      chai
+        .request(app)
+        .get('/?lng=fr')
+        .end((err, res) => {
+          res.should.have.status(200);
+          res.body.message.should.be.equal('Bienvenue sur le Web de Phantom');
+          res.body.error.should.be.equal(false);
+
+          done();
```
Comment (David-Neza) [inline test/test.js:33]: Everything looks great --nice work

**Review by David-Neza — APPROVED** [review]

**Pull request merged**